pyodbc.cpython-310-aarch64-linux-gnu.so,sha256=adEJCfBaay4M3sNSOGQxzEsdGFf9fy0YOW-ELMDa8TM,971856
pyodbc.pyi,sha256=YSHcS8DHzgggA5eKVJuBpvz6RAOSaO39BqTUEl7GA2c,32673
pyodbc-5.2.0.dist-info/top_level.txt,sha256=Rb69vS-SNN_Bfh99vX7eCcvEAJoyzowG3u4R7W36Flw,8
pyodbc-5.2.0.dist-info/RECORD,,
pyodbc-5.2.0.dist-info/LICENSE.txt,sha256=mFnIV3DJG3hmsPdpyTjjbWTQJgFBtSm5R7Tq2WiD6rQ,868
pyodbc-5.2.0.dist-info/METADATA,sha256=CSPn1FnJoZwQCr5qahAMjloJIeVYFsdjtSAMxpLZHVU,2742
pyodbc-5.2.0.dist-info/WHEEL,sha256=IjQRalpktecrnjQ0mlgISGBVe9z7Fv6GdQT0dw4Gf_A,153
